src/common/common-macros-private.h

```diff
@@ -1,3 +1,18 @@
+/*
+ * Copyright 2018-present MongoDB Inc.
```

Comment:
CDRIVER-4044 intends to make uniform the dates in copyright notices by following advice given in DRIVERS-409 and DRIVERS-329.

Recommend preemptively applying the upcoming changes here by using `2009-present` instead (project inception date to present):

```
Copyright 2009-present MongoDB, Inc.
```